int32 buoy_number
int32 center_buoy_x
int32 center_buoy_y
int32 buoy_area
int32 radius_buoy
bool detection_status
